v2/tools/generator/internal/astmodel/std_references.go

```diff
@@ -83,6 +83,11 @@ var (
 	GenRuntimeValidatorInterfaceName = MakeTypeName(GenRuntimeReference, "Validator")
 	GenRuntimeMetaObjectType         = MakeTypeName(GenRuntimeReference, "MetaObject")
 
+	OptionalConfigMapReferenceType     = NewOptionalType(ConfigMapReferenceType)
```

Comment:
minor: Maybe add a section name here to clarify that this isn't part of just the standard `TypeNames - GenRuntime` above?